pkgs/data/icons/tela-circle-icon-theme/default.nix

```diff
@@ -16,13 +17,13 @@ lib.checkListOfEnum "${pname}: color variants" [ "standard" "black" "blue" "brow
 
 stdenvNoCC.mkDerivation rec {
   inherit pname;
-  version = "2022-02-08";
+  version = "2022-03-07";
 
   src = fetchFromGitHub {
     owner = "vinceliuice";
     repo = pname;
     rev = version;
-    sha256 = "08a1jhirvn2x9hhjr0lqqqayhsf446cddapprxpsnsn9q6x2j2gp";
+    sha256 = "vQeWGZmurvT/UQJ1dx6t+ZeKdJ1Oq9TdHBADw64x18g=";
```

Comment:
Why does darwin want ``03zj00jfdysqlyi1kajjlqvasas16325sxxd1ffpqnhdg0fwxyiq``?

Reply:
Maybe because of this https://github.com/NixOS/nix/issues/1931.

Reply:
<details><summary>List of file names differing in case</summary>

```console
$ pwd
/tmp/tmp/Tela-circle-icon-theme-master

$ find | sort --ignore-case | uniq --ignore-case --repeated --all-repeated
./links/16/devices/drive-harddisk-usb.svg
./links/16/devices/drive-harddisk-USB.svg
./links/16/mimetypes/application-vnd.ms-powerpoint.presentation.macroenabled.12.svg
./links/16/mimetypes/application-vnd.ms-powerpoint.presentation.macroEnabled.12.svg
./links/22/devices/drive-harddisk-usb.svg
./links/22/devices/drive-harddisk-USB.svg
./links/24/devices/drive-harddisk-usb.svg
./links/24/devices/drive-harddisk-USB.svg
./links/scalable/apps/caprine.svg
./links/scalable/apps/Caprine.svg
./links/scalable/apps/com.adobe.Flash-Player-Projector.svg
./links/scalable/apps/com.Adobe.Flash-Player-Projector.svg
./links/scalable/apps/com.deepin.Screenshot.svg
./links/scalable/apps/com.deepin.ScreenShot.svg
./links/scalable/apps/com.github.lpereira.hardinfo.svg
./links/scalable/apps/com.github.lpereira.HardInfo.svg
./links/scalable/apps/com.github.thezbyg.gpick.svg
./links/scalable/apps/com.github.thezbyg.Gpick.svg
./links/scalable/apps/com.github.wjaguar.mtpaint.svg
./links/scalable/apps/com.github.wjaguar.mtPaint.svg
./links/scalable/apps/com.gnome.software.svg
./links/scalable/apps/com.gnome.Software.svg
./links/scalable/apps/com.jetbrains.GoLand.svg
./links/scalable/apps/com.Jetbrains.GoLand.svg
./links/scalable/apps/com.simplenote.simplenote.svg
./links/scalable/apps/com.simplenote.Simplenote.svg
./links/scalable/apps/dictionary.svg
./links/scalable/apps/Dictionary.svg
./links/scalable/apps/display-im6.q16.svg
./links/scalable/apps/display-im6.Q16.svg
./links/scalable/apps/etube.svg
./links/scalable/apps/eTube.svg
./links/scalable/apps/foxitreader.svg
./links/scalable/apps/FoxitReader.svg
./links/scalable/apps/freefilesync.svg
./links/scalable/apps/FreeFileSync.svg
./links/scalable/apps/gecrit.svg
./links/scalable/apps/gEcrit.svg
./links/scalable/apps/Gmail-mail.google.com.svg
./links/scalable/apps/GMail-mail.google.com.svg
./links/scalable/apps/gnomebaker.svg
./links/scalable/apps/GnomeBaker.svg
./links/scalable/apps/Gnome-notes.svg
./links/scalable/apps/Gnome-Notes.svg
./links/scalable/apps/gnome-photos.svg
./links/scalable/apps/Gnome-Photos.svg
./links/scalable/apps/grsync.svg
./links/scalable/apps/Grsync.svg
./links/scalable/apps/Grsync.SVG
./links/scalable/apps/imagej.svg
./links/scalable/apps/ImageJ.svg
./links/scalable/apps/io.lmms.Lmms.svg
./links/scalable/apps/io.lmms.LMMS.svg
./links/scalable/apps/labplot2.svg
./links/scalable/apps/LabPlot2.svg
./links/scalable/apps/linssid.svg
./links/scalable/apps/LinSSID.svg
./links/scalable/apps/luckybackup.svg
./links/scalable/apps/luckyBackup.svg
./links/scalable/apps/mintbackup.svg
./links/scalable/apps/mintBackup.svg
./links/scalable/apps/mintnanny.svg
./links/scalable/apps/mintNanny.svg
./links/scalable/apps/mintupdate.svg
./links/scalable/apps/mintUpdate.svg
./links/scalable/apps/mintwelcome.svg
./links/scalable/apps/mintWelcome.svg
./links/scalable/apps/nl.hjdskes.gcolor3.svg
./links/scalable/apps/nl.hjdskes.Gcolor3.svg
./links/scalable/apps/opentodolist.svg
./links/scalable/apps/OpenTodoList.svg
./links/scalable/apps/org.gnome.baobab.svg
./links/scalable/apps/org.gnome.Baobab.svg
./links/scalable/apps/org.gnome.bijiben.svg
./links/scalable/apps/org.gnome.Bijiben.svg
./links/scalable/apps/org.gnome.books.svg
./links/scalable/apps/org.gnome.Books.svg
./links/scalable/apps/org.gnome.builder.svg
./links/scalable/apps/org.gnome.Builder.svg
./links/scalable/apps/org.gnome.calendar.svg
./links/scalable/apps/org.gnome.Calendar.svg
./links/scalable/apps/org.gnome.cheese.svg
./links/scalable/apps/org.gnome.Cheese.svg
./links/scalable/apps/org.gnome.chess.svg
./links/scalable/apps/org.gnome.Chess.svg
./links/scalable/apps/org.gnome.clocks.svg
./links/scalable/apps/org.gnome.Clocks.svg
./links/scalable/apps/org.gnome.documents.svg
./links/scalable/apps/org.gnome.Documents.svg
./links/scalable/apps/org.gnome.epiphany.svg
./links/scalable/apps/org.gnome.Epiphany.svg
./links/scalable/apps/org.gnome.feedreader.svg
./links/scalable/apps/org.gnome.FeedReader.svg
./links/scalable/apps/org.gnome.files.svg
./links/scalable/apps/org.gnome.Files.svg
./links/scalable/apps/org.gnome.five-or-more.svg
./links/scalable/apps/org.gnome.Five-or-more.svg
./links/scalable/apps/org.gnome.four-in-a-row.svg
./links/scalable/apps/org.gnome.Four-in-a-row.svg
./links/scalable/apps/org.gnome.games.svg
./links/scalable/apps/org.gnome.Games.svg
./links/scalable/apps/org.gnome.hitori.svg
./links/scalable/apps/org.gnome.Hitori.svg
./links/scalable/apps/org.gnome.iagno.svg
./links/scalable/apps/org.gnome.Iagno.svg
./links/scalable/apps/org.gnome.lollypop.svg
./links/scalable/apps/org.gnome.Lollypop.svg
./links/scalable/apps/org.gnome.mahjongg.svg
./links/scalable/apps/org.gnome.Mahjongg.svg
./links/scalable/apps/org.gnome.maps.svg
./links/scalable/apps/org.gnome.Maps.svg
./links/scalable/apps/org.gnome.meld.svg
./links/scalable/apps/org.gnome.Meld.svg
./links/scalable/apps/org.gnome.music.svg
./links/scalable/apps/org.gnome.Music.svg
./links/scalable/apps/org.gnome.nautilus.svg
./links/scalable/apps/org.gnome.Nautilus.svg
./links/scalable/apps/org.gnome.photos.svg
./links/scalable/apps/org.gnome.Photos.svg
./links/scalable/apps/org.gnome.powerstats.svg
./links/scalable/apps/org.gnome.PowerStats.svg
./links/scalable/apps/org.gnome.quadrapassel.svg
./links/scalable/apps/org.gnome.Quadrapassel.svg
./links/scalable/apps/org.gnome.software.svg
./links/scalable/apps/org.gnome.Software.svg
./links/scalable/apps/org.gnome.taquin.svg
./links/scalable/apps/org.gnome.Taquin.svg
./links/scalable/apps/org.gnome.tetravex.svg
./links/scalable/apps/org.gnome.Tetravex.svg
./links/scalable/apps/org.gnome.totem.svg
./links/scalable/apps/org.gnome.Totem.svg
./links/scalable/apps/org.gnome.tweaks.svg
./links/scalable/apps/org.gnome.Tweaks.svg
./links/scalable/apps/org.gnome.weather.application.svg
./links/scalable/apps/org.gnome.Weather.Application.svg
./links/scalable/apps/org.gnome.weather.svg
./links/scalable/apps/org.gnome.Weather.svg
./links/scalable/apps/org.hydrogenmusic.hydrogen.svg
./links/scalable/apps/org.hydrogenmusic.Hydrogen.svg
./links/scalable/apps/org.kde.gwenview.svg
./links/scalable/apps/org.kde.Gwenview.svg
./links/scalable/apps/org.kde.ikona.svg
./links/scalable/apps/org.kde.Ikona.svg
./links/scalable/apps/org.kde.plasma.showactivitymanager.svg
./links/scalable/apps/org.kde.plasma.showActivityManager.svg
./links/scalable/apps/org.mozilla.firefox.svg
./links/scalable/apps/org.mozilla.Firefox.svg
./links/scalable/apps/org.xfce.parole.svg
./links/scalable/apps/org.xfce.Parole.svg
./links/scalable/apps/panini.svg
./links/scalable/apps/Panini.svg
./links/scalable/apps/pcsx2.svg
./links/scalable/apps/PCSX2.svg
./links/scalable/apps/pgadmin4.svg
./links/scalable/apps/pgAdmin4.svg
./links/scalable/apps/qmapshack.svg
./links/scalable/apps/QMapShack.svg
./links/scalable/apps/scudcloud.svg
./links/scalable/apps/ScudCloud.svg
./links/scalable/apps/spotify.svg
./links/scalable/apps/Spotify.svg
./links/scalable/apps/sylpheed.svg
./links/scalable/apps/Sylpheed.svg
./links/scalable/apps/terminal-tango.svg
./links/scalable/apps/Terminal-tango.svg
./links/scalable/apps/vbox.svg
./links/scalable/apps/Vbox.svg
./links/scalable/apps/VBox.svg
./links/scalable/apps/wmail.svg
./links/scalable/apps/WMail.svg
./links/scalable/apps/Youtube-youtube.com.svg
./links/scalable/apps/YouTube-youtube.com.svg
./links/scalable/mimetypes/application-vnd.ms-powerpoint.presentation.macroenabled.12.svg
./links/scalable/mimetypes/application-vnd.ms-powerpoint.presentation.macroEnabled.12.svg
./links/symbolic/apps/org.gnome.gedit-symbolic.svg
./links/symbolic/apps/org.gnome.Gedit-symbolic.svg
./links/symbolic/apps/org.gnome.tweaks-symbolic.svg
./links/symbolic/apps/org.gnome.Tweaks-symbolic.svg
```

</details>

Reply:
Fixed by restricting platforms to linux, as darwin systems use case-insensitive filesystems that cause hash mismatches.